update-framebuffer.sh

```diff
@@ -43,8 +43,6 @@ if [[ ! -f /opt/kubesail/pibox-framebuffer ]]; then
   ln -vs $FB_PATH /opt/kubesail/pibox-framebuffer
 fi
 
-chown -R 989 /opt/kubesail/ || true
```

Comment:
This also needs to be left in place (the `kubesail-agent` mounts `/opt/kubesail`, which requires a known UID).